packages/react-native-editor/CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ For each user feature we should also add a importance categorization label  to i
 -   [*] Add enableCaching param to fetch request on Android [#31186]
 -   [***] Add reusable blocks to the inserter menu. [#28495]
 -   [*] The BottomSheet Cell component now supports the help prop so that a hint can be supplied to all Cell based components. [#30885]
-
+-   [*] Audio block: Add Insert from URL functionality. [#27817]
```

Comment:
I think this entry should go to the `Unreleased` section since release `1.53.0` was already cut.